tutorials/tts/Fastpitch_Training_GermanTTS.ipynb

```diff
@@ -0,0 +1,1158 @@
+    "2. `duration`: duration of the wav file\n",
+    "3. `text`: original text supplied by OpenSLR\n",
+    "    \n",
+    "After that, the script randomly splits the datapoints into 3 buckets, `train_manifest.json`, `val_manifest.json` and `test_manifest.json`. Example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f92ea2e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!(cd /NeMo && \\\n",
+    "    python get_data.py \\\n",
+    "        --data-root /Data/ \\\n",
+    "        --val-size 0.1 \\\n",
+    "        --test-size 0.2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37265dd9",
+   "metadata": {},
+   "source": [
+    "In the example above, 10% datapoints go to validation set, 20% go to test set and the remaining 70% go to training set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19a4e224",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# /Data directory looks like\n",
+    "!ls /Data -R"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c89e1a7f",
+   "metadata": {},
+   "source": [
+    "```bash\n",
+    "$ ls /Data -R\n",
+    "/Data:\n",
+    "thorsten-de\n",
+    "thorsten-de_v02.tgz\n",
+    "\n",
+    "/Data/thorsten-de:\n",
+    "metadata.csv\n",
+    "metadata_shuf.csv\n",
+    "metadata_train.csv\n",
+    "metadata_val.csv\n",
+    "test_manifest.json\n",
+    "train_manifest.json\n",
+    "val_manifest.json\n",
+    "wavs\n",
+    "\n",
+    "/Data/thorsten-de/wavs:\n",
+    "00025a6fbea659dae6ece011e749aa34.wav\n",
+    "000314280388fb390b3e70b69ee53a23.wav\n",
+    "000624f768d7e282534a850980619fb2.wav\n",
+    "...\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "019f051c",
+   "metadata": {},
+   "source": [
+    "## 3. Normalizing text\n",
+    "\n",
+    "The script above, i.e. `scripts/dataset_processing/tts/openslr/get_data.py`, also generates a another field per each datapoint:\n",
+    "- `normalized_text`: normalized text via NeMo's text normalizer:\n",
+    "    ```python\n",
+    "    nemo_text_processing.text_normalization.normalize.Normalizer(lang=\"de\", input_case=\"cased\", overwrite_cache=True, cache_dir=str(file_path / \"cache_dir\"))\n",
+    "    ```\n",
+    "    \n",
+    "German language text normalizer (defined here: `nemo_text_processing/text_normalization/de`) was created using the tutorial shared under NeMo's `Grammar customization` documentation [here](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/text_normalization/wfst/wfst_text_normalization.html#grammar-customization). Here are some example records:\n",
+    "```json\n",
+    "{'audio_filepath': '/Data/thorsten-de/wavs/f1becc89cb4079a123ead68c9c8bb8ae.wav', 'duration': 7.250023, 'text': 'Öffne den Webbrowser und rufe www.archlinux.org auf.', 'normalized_text': 'Öffne den Webbrowser und rufe w w w punkt a r c h l i n u x punkt o r g auf.'}\n",
```

Comment:
May want to switch to double quotes here, I think the JSON parser is freaking out.